joeynmt/training.py

```diff
@@ -103,11 +108,12 @@ def __init__(self, model: Model, config: dict) -> None:
         if self.early_stopping_metric in ["ppl", "loss"]:
             self.minimize_metric = True
         elif self.early_stopping_metric == "eval_metric":
-            if self.eval_metric in ["bleu", "chrf"]:
-                self.minimize_metric = False
-            # eval metric that has to get minimized (not yet implemented)
-            else:
-                self.minimize_metric = True
+            for eval_metric in self.eval_metrics:
```

Comment:
This is ambiguous: for early stopping, we actually have to pick one of the metrics, and minimize_metric has to be set according to that one. So I propose we take by default the first evaluation metric. That would just need documentation and instead of looping over the metrics here, we would only take the first.